components/raftstore/src/store/util.rs

```diff
@@ -685,6 +686,7 @@ pub(crate) fn u64_to_timespec(u: u64) -> Timespec {
 pub fn parse_data_at<T: Message + Default>(data: &[u8], index: u64, tag: &str) -> T {
     let mut result = T::default();
     result.merge_from_bytes(data).unwrap_or_else(|e| {
+        error!("meet corrupted raft message: {}", log_wrappers::Value::value(data));
```

Comment:
How about also making it part of panic message?

Reply:
I'm not sure if it's a good idea, because the raft log body can be very big which may make it hard to see the panic info.

Reply:
There is no difference. Either panic message hard to read or the log is hard to read. Not putting it into panic message will require additional operation to search logs.